src/v3/src/components/Link/Link.tsx

```diff
@@ -69,6 +69,8 @@ const Link: UISchemaElementComponent<{
         <LinkMui
           href={href}
           ref={focusRef}
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(dataSe && { 'data-se': dataSe } )}
```

Comment:
What happens if you don't check whether dataSe is set? My guess is the prop would be ignored?

Reply:
yes i believe react ignores attributes that are undefined

Reply:
I think if you dont check the truthy-ness of the variable, it will still set it as a prop on the element, but without a value

so it would look something like 

```
<a href="..." data-se>Blah</a>
```

Correct me if I am wrong though but I _thought_ that is what happens 🤔

Reply:
https://codesandbox.io/s/sweet-shape-d15j7z?file=/src/App.js

Reply:
Sure enough it ignores it 🤦 

Ty! So really we could clean all of these up and get rid of that condition (at a later time, but good to know! TY!)